motion/thruster_interface_auv_cpp/src/thruster_interface_auv_ros.cpp

```diff
@@ -0,0 +1,77 @@
+#include "thruster_interface_auv_ros.hpp"
+
+ThrusterInterfaceAUVNode::ThrusterInterfaceAUVNode() : Node("thruster_interface_auv_node") {
+
+    //create a subscriber that takes data from thruster forces and publisher for debugging
```

Comment:
Name of subscriber and publisher should be exposed as a ros param and read from a config file